src/ui/summary-ui-handler.ts

```diff
@@ -275,6 +289,9 @@ export default class SummaryUiHandler extends UiHandler {
       this.splicedIcon.on('pointerout', () => (this.scene as BattleScene).ui.hideTooltip());
     }
 
+    if(this.scene.gameData.starterData[this.pokemon.species.getRootSpeciesId()].classicWinCount > 0)
```

Comment:
Bit of an edge case here. I think it makes sense to check both root species.
```suggestion
    if(this.scene.gameData.starterData[this.pokemon.species.getRootSpeciesId()].classicWinCount > 0 && this.scene.gameData.starterData[this.pokemon.species.getRootSpeciesId(true)].classicWinCount > 0)
```